app/models/claim.rb

```diff
@@ -253,7 +253,13 @@ def submittable?
   end
 
   def approvable?
-    submitted? && !held? && !payroll_gender_missing? && (!decision_made? || awaiting_qa?) && !payment_prevented_by_other_claims? && attributes_flagged_by_risk_indicator.none?
+    submitted? &&
+      !held? &&
+      !payroll_gender_missing? &&
+      (!decision_made? || awaiting_qa?) &&
+      !payment_prevented_by_other_claims? &&
+      attributes_flagged_by_risk_indicator.none? &&
+      policy_specific_approvable?
```

Comment:
Not a blocker but `Claim#policy_specific_approvable?` and `Eligibility#approvable?` might be better as a method on the policy:

```rb
module Policies::EarlyYearsPayments
  def approvable?(claim)
    claim.tasks.find_or_initialize_by(name: "employment").passed?
  end
end

module BasePolicy
  def approvable?(claim)
    true
  end
end
```

that way the policy has all the policy specific rules.

```suggestion
    submitted? &&
      !held? &&
      !payroll_gender_missing? &&
      (!decision_made? || awaiting_qa?) &&
      !payment_prevented_by_other_claims? &&
      attributes_flagged_by_risk_indicator.none? &&
      policy.approvable?(self)
```

Reply:
As part of the LUPEALPHA-1202 ticket I've added a method like this to the policy to determine if a task is available to admins, and seems to work quite well / keeps the policy specific rules in the policy class